src/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
```

Comment:
Since we are using react-scripts there is no need to manually declare this module. The react-app-env.d.ts is a more complete approach.